contextMenu.css

```diff
@@ -64,6 +64,6 @@
   height: 20px;
   left: 0px;
   top: 3px;
-  position: absolute;
```

Comment:
That's just a typo. I'm working on two repositories and got thing messed up. It shouldn't be removed.